bundles.config.js

```diff
@@ -26,158 +26,430 @@ const TYPES = exports.TYPES = {
 };
 
 exports.extensionBundles = [
-  {name: 'amp-3d-gltf', version: '0.1', type: TYPES.MEDIA},
-  {name: 'amp-3q-player', version: '0.1', type: TYPES.MEDIA},
+  {
+    name: 'amp-3d-gltf',
+    version: '0.1',
+    latestVersion: '0.1',
```

Comment:
Nit: if `version` is a string (not an array), why require this?

Reply:
Because the whole purpose of this is to guard against someone adding a new version and having `-latest` automatically move forward. This way, they'd have to explicitly update both fields in order to change the latest version.

I could go either way on this. @aghassemi your thoughts?

Reply:
We would throw when it's an array and not set.

Reply:
Done in 726a1ed32a282e31cf093030f7dfa0fbc87555ac.

Reply:
As @aghassemi points out in https://github.com/ampproject/amphtml/pull/21190#discussion_r261453944, this won't work when there are two separate entries for an extension.